### DISCUSSION TITLE
Part 2 - TK Results

### DISCUSSION THREAD

**Discussion by Thessi** [sebastian-hofstaetter/teaching#105] [Advanced IR: Exercise Q&A] [answered]
Hey,

according to the assignment, the TK results should improve over CONV-KNRM. We implemented TK and are not seeing any improvements, it's actually quite a bit worse (even after playing around with weights).
The assignment also mentions that the scores will be worse due to a smaller amount of data.
Are we still expected to see an improvement? Or is this just a theoretical improvement, in case we had perfect conditions?

BR

**Comment by sophiaalthammer** [comment, marked as answer]
Hello,

normally TK should outperform Conv-KNRM, but if this is not the case, you can just report in your report on the experiments you did and the hyperparameter you tried out! As long as you demonstrate that you did different experinments and investigated different hyperparameters, that fine!

Best,
Sophia
